pymc/gp/hsgp_approx.py

```diff
@@ -85,6 +89,82 @@ def calc_basis_periodic(
     return phi_cos, phi_sin
 
 
+class HSGPParams(NamedTuple):
+    m: int
+    c: float
+    S: float
+
+
+def approx_hsgp_hyperparams(
+    x: np.ndarray, lengthscale_range: list[float], cov_func: str
+) -> HSGPParams:
+    """Utility function that uses heuristics to recommend minimum `m` and `c` values,
+    based on recommendations from Ruitort-Mayol et. al.
+
+    In practice, you need to choose `c` large enough to handle the largest lengthscales,
+    and `m` large enough to accommodate the smallest lengthscales.
+
+    NB: These recommendations are based on a one-dimensional GP.
+
+    Parameters
+    ----------
+    x_range : List[float]
```

Comment:
just noticed that this bit of the docstring is out of date.  looks like the function takes x instead of x_range now

Reply:
Since it's x now, it'd be good to remind people in the docstring that this `x` should be the x values you'll want to predict over, not just the training x.

Reply:
Good catch @bwengals ! Just pushed the change. Will merge once it's all green